src/libcore/macros.rs

```diff
@@ -300,6 +300,7 @@ macro_rules! debug_assert_ne {
 ///     Ok(())
 /// }
 /// ```
+#[deprecated(since = "1.38.0", note = "the `?` operator should be used instead")]
```

Comment:
```suggestion
#[deprecated(since = "1.37.0", note = "use the `?` operator instead")]
```
I'm hoping we can land this before we branch master.